model/id.go

```diff
@@ -24,3 +25,17 @@ func NewID() string {
 	b.Truncate(26) // removes the '==' padding
 	return b.String()
 }
+
+// ClusterNewID is a globally unique identifier for cluster ID.  It is a [A-Z0-9] string 26
+func ClusterNewID() string {
+	var b bytes.Buffer
+	encoder := base32.NewEncoder(encoding, &b)
+	encoder.Write(uuid.NewRandom())
+	encoder.Close()
+	b.Truncate(26) // removes the '==' padding
+	r := rune(b.String()[0])
+	if unicode.IsNumber(r) {
+		return ClusterNewID()
+	}
```

Comment:
I would like to see this part of the logic changed to just generate a random letter and replace the first character of the ID if it was a number. In theory, this function could get "unlucky" which would result in it calling itself many times in a row. That said, this will rarely be used so I won't block the PR.

Reply:
@gabrieljackson fair point, i'have changed the code to replace the recursion with single random first char concatenation.